addons/crm/static/src/new/activity_menu_patch.js

```diff
@@ -0,0 +1,36 @@
+/* @odoo-module */
+
+import { ActivityMenu } from "@mail/new/web/activity/activity_menu";
+import { patch } from "@web/core/utils/patch";
+
+patch(ActivityMenu.prototype, "crm", {
+    availableViews(group) {
+        if (group.model === "crm.lead") {
```

Comment:
Isn't it structurally impossible to reach this case if `openActivityGroup` is completely overridden with the same condition?

Reply:
Actually there are 2 cases from old code: `onClick` and `onClickFilterButton`.

Reply:
`onClickFilterButton` is click on the whole item apparently, whereas `onClick` is on a specific action button of that item.

Reply:
New code
```xml
                                    <t t-foreach="group.actions or []" t-as="action" t-key="action_index">
                                        <button class="btn fa p-0" type="button" t-att-title="action.name" t-att-class="action.icon"/>
                                    </t>
```
it looks like this is missing its click handler

Reply:
It sucks a bit, but it works when `crm_enterprise` has precedence over `crm`. It's all about whether enterprise is installed or not.

I'm adding a `import @crm/new/activity_menu` in `crm_enterprise` to ensure the order is `crm` then `crm_enterprise`, so that `crm_enterprise` has precedence over `crm`.